templates/accounts/donation_list.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+<div id="content_full"> <!-- START of Content area -->
+
+  <h1>Donations</h1>
+   <h2>Last Donations</h2>
+    <div class="donations-list">
+    {% for donation in object_list %}
+      <div class="donation-item">
+        <div class="donation-item-left"><img src="/media/images/70x70_avatar.png"></div>
+        <div class="donation-item-right"><p>{{donation.display_name}} <br /> {{donation.amount}} {{donation.currency}}</p></div>
+      </div>
+    {% endfor %}
+    </div>
+    {% if is_paginated %}
```

Comment:
@andrebola is doing pagination using a `ListView` which does not use our typical pagination strategy with our [templatetag](https://github.com/MTG/freesound/blob/4e4b91fa45adabb5294487eeed384596425fd8bb/general/templatetags/paginator.py) and [pagination](https://github.com/MTG/freesound/blob/4e4b91fa45adabb5294487eeed384596425fd8bb/utils/pagination.py) function. 
I think it's good if we do things here in the officially proposed "Django way". A number of other simple views that we have could in fact be refactored similarly though no need for that now.

Nevertheless we should make the "new" paginator look the same as the old one (show the numbers etc), I'm sure there are options and ways to do it.